nano/node/active_transactions.cpp

```diff
@@ -747,6 +747,8 @@ void nano::active_transactions::add_inactive_votes_cache (nano::unique_lock<nano
 				inactive_by_arrival.erase (inactive_by_arrival.begin ());
 			}
 		}
+
+		node.stats.inc (nano::stat::type::vote_cache, nano::stat::detail::vote_new);
```

Comment:
Is this increment at the right place? It appears that it is counting both existing and new votes.

Reply:
Right, this stat should be called vote_processed, because we want to count every vote processed.